Expensify/App
# Afonseca dismiss hamburger

fixes: https://github.com/Expensify/ReactNativeChat/issues/148

### Problem 
We should allow the user to dismiss the hamburger menu when the user taps out of the menu

### Tests
1. Verify that the chat works as expected, it scrolls and you can add a new comment
1. Open Hamburger menu and select another chat, Verify that it opens the other chat
1. Open Hamburger menu and tap outside it, on the white part on the right side.
1. Verify that the Hamburger menu is dismissed.
